nbs/20_comparison_others/py/50-combine_all.py

```diff
@@ -0,0 +1,296 @@
+display(COMPARISONS_DIR)
+
+# %% tags=[]
+INPUT_DIR = COMPARISONS_DIR / METHOD_NAME
+display(INPUT_DIR)
+
+# %% tags=[]
+INPUT_CORR_FILE_TEMPLATE = (
+    DATASET_CONFIG["SIMILARITY_MATRICES_DIR"]
+    / DATASET_CONFIG["SIMILARITY_MATRIX_FILENAME_TEMPLATE"]
+)
+display(INPUT_CORR_FILE_TEMPLATE)
+
+# %% tags=[]
+INPUT_CORR_FILE_TEMPLATE = (
+    DATASET_CONFIG["SIMILARITY_MATRICES_DIR"]
+    / DATASET_CONFIG["SIMILARITY_MATRIX_FILENAME_TEMPLATE"]
+)
+display(INPUT_CORR_FILE_TEMPLATE)
+
+# %% tags=[]
+OUTPUT_FILE = (
+    COMPARISONS_DIR / f"gtex_v8_data_{GTEX_TISSUE}-{GENE_SEL_STRATEGY}-all.pkl"
+)
+display(OUTPUT_FILE)
+
+# %% [markdown] tags=[]
+# # Load data
+
+# %% [markdown] tags=[]
+# ## Clustermatch
+
+# %% tags=[]
+clustermatch_df = pd.read_pickle(
+    str(INPUT_CORR_FILE_TEMPLATE).format(
+        tissue=GTEX_TISSUE,
+        gene_sel_strategy=GENE_SEL_STRATEGY,
+        corr_method="clustermatch",
+    )
+)
+
+# %% tags=[]
+clustermatch_df.shape
+
+# %% tags=[]
+clustermatch_df.head()
+
+# %% tags=[]
+clustermatch_df = get_upper_triag(clustermatch_df)
+
+# %% tags=[]
+clustermatch_df = (
+    clustermatch_df.unstack()
+    .rename_axis((None, None))
+    .dropna()
+    .sort_index()
+    .rename("clustermatch")
+)
+
+# %% tags=[]
+clustermatch_df.shape
+
+# %% tags=[]
+clustermatch_df.head()
+
+# %% [markdown] tags=[]
+# ## Pearson
+
+# %% tags=[]
+pearson_df = pd.read_pickle(
+    str(INPUT_CORR_FILE_TEMPLATE).format(
+        tissue=GTEX_TISSUE,
+        gene_sel_strategy=GENE_SEL_STRATEGY,
+        corr_method="pearson",
+    )
+)
+
+# %% tags=[]
+pearson_df.shape
+
+# %% tags=[]
+pearson_df.head()
+
+# %% tags=[]
+pearson_df = get_upper_triag(pearson_df)
+
+# %% tags=[]
+# make pearson abs
+pearson_df = (
+    pearson_df.unstack()
+    .rename_axis((None, None))
+    .dropna()
+    .abs()
+    .sort_index()
+    .rename("pearson")
+)
+
+# %% tags=[]
+pearson_df.shape
+
+# %% tags=[]
+pearson_df.head()
+
+# %% [markdown] tags=[]
+# ## Spearman
+
+# %% tags=[]
+spearman_df = pd.read_pickle(
+    str(INPUT_CORR_FILE_TEMPLATE).format(
+        tissue=GTEX_TISSUE,
+        gene_sel_strategy=GENE_SEL_STRATEGY,
+        corr_method="spearman",
+    )
+)
+
+# %% tags=[]
+spearman_df.shape
+
+# %% tags=[]
+spearman_df.head()
+
+# %% tags=[]
+spearman_df = get_upper_triag(spearman_df)
+
+# %% tags=[]
+# make spearman abs
+spearman_df = (
+    spearman_df.unstack()
+    .rename_axis((None, None))
+    .dropna()
+    .abs()
+    .sort_index()
+    .rename("spearman")
+)
+
+# %% tags=[]
+spearman_df.shape
+
+# %% tags=[]
+spearman_df.head()
+
+# %% [markdown] tags=[]
+# ## MIC
+
+# %% tags=[]
+mic_all_df = (
+    pd.read_pickle(
+        INPUT_DIR
+        / f"gtex_v8_data_{GTEX_TISSUE}-{GENE_SEL_STRATEGY}-all-gene_pairs-sample_0-mic.pkl"
+    )
+    .rename("mic")
+    .to_frame()
+)
+
+# %% tags=[]
+mic_all_df["mic_subset"] = "all"
+
+# %% tags=[]
+mic_all_df.shape
+
+# %% tags=[]
```

Comment:
Do you want to still have these commented blocks in the code?

Reply:
I would like to keep them for now, because I might need to do something with those stratified samples.